src/notify/subscription/db.go

```diff
@@ -42,13 +37,12 @@ func get(id string) (Subscription, error) {
 	defer rows.Close()
 
 	if rows.Next() {
-		err := scan(rows, &subscription)
-		if err != nil {
+		if err := scan(rows, &subscription); err != nil {
 			return subscription, err
 		}
 	}
 
-	return subscription, nil
+	return subscription, errors.New("No subscription found")
```

Comment:
Also missing a check for rows.Error().  See https://golang.org/pkg/database/sql/#DB.Query

All of this goes away with QueryRow() :-)  See event's lookup().